src/web_interface/components/database_routes.py

```diff
@@ -160,7 +159,8 @@ def _build_search_query(self):
         return json.dumps(query)
 
     def _add_hash_query_to_query(self, query, value):
-        hash_types = parse_comma_separated_list(getattr(cfg, 'file_hashes')['hashes'])
+        # FIXME: The frontend should not need to know how the plugin is configured
+        hash_types = ['md5', 'sha1', 'sha256', 'sha512', 'ripemd160', 'whirlpool']
```

Comment:
how do we fix this? Hard coding is probably even worse :sweat_smile:

Reply:
No idea. Previously the frontend just loaded the config of the plugin, which imo makes no sense (but works).
Hardcoding this is probably fine as this is only used in the basic search.
So if e.g. the hash plugin now supports more hashes the user could still use the advanced search to search for them.

Reply:
We would probably need to use the intercom to implement a "clean" way to do this. Let's keep it like it is for the time being.